### ISSUE TITLE
asthmatic bundling support

### ISSUE THREAD

**Issue by devmondo** [stealjs/steal-tools#516]
hi, 

is there support for this https://github.com/systemjs/builder#bundle-arithmetic

specially this example 

`builder.bundle('app/**/* - [app/**/*]', 'dependencies.js', { minify: true, sourceMaps: true });`

this allows to create 2 output, one the app code it self and the other is the vendor libs.

thanks in advanced.

**Comment by matthewp** [comment]
Yes and no. You can really do anything you want, including customizing the bundling, see the [streams API](http://stealjs.com/docs/steal-tools.guides.streams.html). Doing what you want to do here will take a bit of work and it's not particularly well documented, but in your case you'll want to exclude the [bundle stream](http://stealjs.com/docs/steal-tools.streams.bundle.html) and provide one yourself.

There's not a higher level API to customize the bundle outputs. For the most part it's not needed as steal-tools smartly bundles resources in an optimized fashion.  *Maybe* there could be another high-level API that gives you equivalent customizability but it doesn't exist yet and is not on my radar to do (I'd rather improve our plugins API and work on general performance improvements).

**Comment by devmondo** [comment]
thanks for the detailed reply, indeed it is well documented, but as you said it takes some work, looking forward for similar implementation like JSPM.
